### PR TITLE
[WIP] Use Knative Status package.

### PR DESCRIPTION
**Pull request by jmprusi**

### CODE REVIEW

File: pkg/envoy/route.go [hunk 1]
```diff
@@ -17,6 +17,10 @@ limitations under the License.
 package envoy
 
 import (
```
Comment (mattmoor) [inline pkg/envoy/route.go]: Format Go code:
```suggestion
import (
	"net/http"
	"time"

```

File: pkg/envoy/route.go [hunk 2]
```diff
@@ -17,6 +17,10 @@ limitations under the License.
...
+	externalAuthzService "github.com/envoyproxy/go-control-plane/envoy/config/filter/http/ext_authz/v2"
+	"github.com/envoyproxy/go-control-plane/pkg/wellknown"
+	"github.com/gogo/protobuf/proto"
+	"github.com/golang/protobuf/ptypes/any"
 	"net/http"
 	"time"
```
Comment (mattmoor) [inline pkg/envoy/route.go]: Format Go code:
```suggestion

```

File: pkg/generator/ingress_translator.go [hunk 3]
```diff
@@ -18,6 +18,8 @@ package generator
 
 import (
 	"fmt"
```
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion
	"fmt"
	"time"

```

File: pkg/generator/ingress_translator.go [hunk 4]
```diff
@@ -18,6 +18,8 @@ package generator
...
+	"github.com/google/uuid"
+	knativeIngress "knative.dev/serving/pkg/network/ingress"
 	"time"
```
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion

```

File: pkg/generator/ingress_translator.go [hunk 5]
```diff
@@ -230,6 +239,16 @@ func lbEndpointsForKubeEndpoints(kubeEndpoints *kubev1.Endpoints, targetPort int
...
 }
 
+func createStatusRouteForIngress(ingress *v1alpha1.Ingress, extAuthzEnabled bool) *route.Route {
+	hash, _ := knativeIngress.ComputeHash(ingress)
+	ingressKey := fmt.Sprintf("%x", hash)
+	uuidVar, _ := uuid.NewUUID()
+	routeName := uuidVar.String()
+	readyPath := "/ready_" + ingressKey
+	return envoy.NewRouteReadiness(routeName, readyPath,
+		map[string]string{"KOURIER-PROBER":"ready"}, extAuthzEnabled)
```
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion
		map[string]string{"KOURIER-PROBER": "ready"}, extAuthzEnabled)
```

File: pkg/reconciler/ingress/ingress.go [hunk 6]
```diff
@@ -19,11 +19,10 @@ package ingress
 import (
 	"context"
 	"fmt"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go]: Format Go code:
```suggestion
	"fmt"
	"reflect"

```

File: pkg/reconciler/ingress/ingress.go [hunk 7]
```diff
@@ -19,11 +19,10 @@ package ingress
...
-	"reflect"
-
 	"github.com/3scale/kourier/pkg/envoy"
 	"github.com/3scale/kourier/pkg/generator"
 	"github.com/3scale/kourier/pkg/knative"
+	"reflect"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go]: Format Go code:
```suggestion

```

File: pkg/envoy/route.go (hunk 1 above)
Comment (mattmoor) [inline pkg/envoy/route.go]: Format Go code:
```suggestion
import (
	"net/http"
	"time"

```

File: pkg/envoy/route.go (hunk 2 above)
Comment (mattmoor) [inline pkg/envoy/route.go]: Format Go code:
```suggestion

```

File: pkg/generator/ingress_translator.go (hunk 3 above)
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion
	"fmt"
	"time"

```

File: pkg/generator/ingress_translator.go (hunk 4 above)
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/ingress.go (hunk 6 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go]: Format Go code:
```suggestion
	"fmt"
	"reflect"

```

File: pkg/reconciler/ingress/ingress.go (hunk 7 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go]: Format Go code:
```suggestion

```

File: pkg/envoy/route.go (hunk 1 above)
Comment (mattmoor) [inline pkg/envoy/route.go]: Format Go code:
```suggestion
import (
	"net/http"
	"time"

```

File: pkg/envoy/route.go (hunk 2 above)
Comment (mattmoor) [inline pkg/envoy/route.go]: Format Go code:
```suggestion

```

File: pkg/generator/ingress_translator.go (hunk 3 above)
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion
	"fmt"
	"time"

```

File: pkg/generator/ingress_translator.go (hunk 4 above)
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion

```

File: copy_ksvc.sh [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+      containers:
+      - image: gcr.io/knative-samples/helloworld-go
+        env:
+        - name: TARGET
+          value: "Go Sample v1"
+EOF
+}
+
+function ksvc_delete {
+
```
Comment (mattmoor) [inline copy_ksvc.sh]: Remove trailing whitespace:
```suggestion

```

File: copy_ksvc.sh [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+function ksvc_wait_for_readiness {
+	while kubectl get ksvc -n $namespace $1 -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}' | grep -q -v True; do sleep 1; done
+}
+
+function parse_ready_time {
+	creation=$(kubectl get ksvc -n $namespace $1 -o jsonpath='{.metadata.creationTimestamp}')
+	readiness=$(kubectl get ksvc -n $namespace $1 -o jsonpath='{.status.conditions[?(@.type=="Ready")].lastTransitionTime}')
+
+	revision=$(kubectl get ksvc -n $namespace $1 -o jsonpath='{.status.latestCreatedRevisionName}')
+
```
Comment (mattmoor) [inline copy_ksvc.sh]: Remove trailing whitespace:
```suggestion

```

File: copy_ksvc.sh [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+function check_curl {
+	url=$(kubectl get ksvc -n $namespace $1 -o jsonpath='{.status.url}')
+
+	if [ -n "$url" ]
+	then
+	        url=$(kubectl get ksvc -n $namespace $1 -o jsonpath='{.status.url}')
+	fi
+
```
Comment (mattmoor) [inline copy_ksvc.sh]: Remove trailing whitespace:
```suggestion

```

File: curl.stderr [hunk 11]
```diff
@@ -0,0 +1,18 @@
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 127.0.0.1...
+* TCP_NODELAY set
+* Connected to testfoo56.default.127.0.0.1.nip.io (127.0.0.1) port 80 (#0)
+> GET / HTTP/1.1
```
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
> GET / HTTP/1.1
```

File: curl.stderr [hunk 12]
```diff
@@ -0,0 +1,18 @@
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 127.0.0.1...
+* TCP_NODELAY set
+* Connected to testfoo56.default.127.0.0.1.nip.io (127.0.0.1) port 80 (#0)
+> GET / HTTP/1.1
+> Host: testfoo56.default.127.0.0.1.nip.io
```
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
> Host: testfoo56.default.127.0.0.1.nip.io
```

File: curl.stderr [hunk 13]
```diff
@@ -0,0 +1,18 @@
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 127.0.0.1...
+* TCP_NODELAY set
+* Connected to testfoo56.default.127.0.0.1.nip.io (127.0.0.1) port 80 (#0)
+> GET / HTTP/1.1
+> Host: testfoo56.default.127.0.0.1.nip.io
+> User-Agent: curl/7.64.1
```
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
> User-Agent: curl/7.64.1
```

File: curl.stderr [hunk 14]
```diff
@@ -0,0 +1,18 @@
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 127.0.0.1...
+* TCP_NODELAY set
+* Connected to testfoo56.default.127.0.0.1.nip.io (127.0.0.1) port 80 (#0)
+> GET / HTTP/1.1
+> Host: testfoo56.default.127.0.0.1.nip.io
+> User-Agent: curl/7.64.1
+> Accept: */*
```
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
> Accept: */*
```

File: curl.stderr [hunk 15]
```diff
@@ -0,0 +1,18 @@
...
+                                 Dload  Upload   Total   Spent    Left  Speed
+
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 127.0.0.1...
+* TCP_NODELAY set
+* Connected to testfoo56.default.127.0.0.1.nip.io (127.0.0.1) port 80 (#0)
+> GET / HTTP/1.1
+> Host: testfoo56.default.127.0.0.1.nip.io
+> User-Agent: curl/7.64.1
+> Accept: */*
+>
```
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
>
```

File: curl.stderr [hunk 16]
```diff
@@ -0,0 +1,18 @@
...
+
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 127.0.0.1...
+* TCP_NODELAY set
+* Connected to testfoo56.default.127.0.0.1.nip.io (127.0.0.1) port 80 (#0)
+> GET / HTTP/1.1
+> Host: testfoo56.default.127.0.0.1.nip.io
+> User-Agent: curl/7.64.1
+> Accept: */*
+> 
+< HTTP/1.1 404 Not Found
```
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
< HTTP/1.1 404 Not Found
```

File: curl.stderr [hunk 17]
```diff
@@ -0,0 +1,18 @@
...
+< date: Sun, 29 Mar 2020 19:41:55 GMT
```
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
< date: Sun, 29 Mar 2020 19:41:55 GMT
```

File: curl.stderr [hunk 18]
```diff
@@ -0,0 +1,18 @@
...
+< server: envoy
```
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
< server: envoy
```

File: curl.stderr [hunk 19]
```diff
@@ -0,0 +1,18 @@
...
+< content-length: 0
```
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
< content-length: 0
```

File: curl.stderr [hunk 20]
```diff
@@ -0,0 +1,18 @@
...
+<
```
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
<
```

File: ready_timings.sh [hunk 21]
```diff
@@ -0,0 +1,92 @@
...
+function ksvc_wait_for_readiness {
+	while kubectl get ksvc -n $namespace $1 -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}' | grep -q -v True; do sleep 1; done
+}
+
+function parse_ready_time {
+	creation=$(kubectl get ksvc -n $namespace $1 -o jsonpath='{.metadata.creationTimestamp}')
+	readiness=$(kubectl get ksvc -n $namespace $1 -o jsonpath='{.status.conditions[?(@.type=="Ready")].lastTransitionTime}')
+
+	revision=$(kubectl get ksvc -n $namespace $1 -o jsonpath='{.status.latestCreatedRevisionName}')
+
```
Comment (mattmoor) [inline ready_timings.sh]: Remove trailing whitespace:
```suggestion

```

File: ready_timings.sh [hunk 22]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+function check_curl {
+	url=$(kubectl get ksvc -n $namespace $1 -o jsonpath='{.status.url}')
+
+	if [ -n "$url" ]
+	then
+	        url=$(kubectl get ksvc -n $namespace $1 -o jsonpath='{.status.url}')
+	fi
+
```
Comment (mattmoor) [inline ready_timings.sh]: Remove trailing whitespace:
```suggestion

```

File: test.sh [hunk 23]
```diff
@@ -0,0 +1,82 @@
...
+function ksvc_wait_for_readiness {
+	while kubectl get ksvc -n $namespace $1 -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}' | grep -q -v True; do sleep 1; done
+}
+
+function parse_ready_time {
+	creation=$(kubectl get ksvc -n $namespace $1 -o jsonpath='{.metadata.creationTimestamp}')
+	readiness=$(kubectl get ksvc -n $namespace $1 -o jsonpath='{.status.conditions[?(@.type=="Ready")].lastTransitionTime}')
+
+	revision=$(kubectl get ksvc -n $namespace $1 -o jsonpath='{.status.latestCreatedRevisionName}')
+
```
Comment (mattmoor) [inline test.sh]: Remove trailing whitespace:
```suggestion

```

File: pkg/generator/ingress_translator.go [hunk 24]
```diff
@@ -18,6 +18,7 @@ package generator
 
 import (
 	"fmt"
```
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion
	"fmt"
	"time"

```

File: pkg/generator/ingress_translator.go [hunk 25]
```diff
@@ -18,6 +18,7 @@ package generator
...
+	ingress2 "knative.dev/serving/pkg/network/ingress"
 	"time"
```
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/controller.go [hunk 26]
```diff
@@ -18,6 +18,8 @@ package ingress
 
 import (
 	"context"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go]: Format Go code:
```suggestion
	"context"
	"strings"
	"time"

```

File: pkg/reconciler/ingress/controller.go [hunk 27]
```diff
@@ -18,6 +18,8 @@ package ingress
...
+	"knative.dev/pkg/reconciler"
+	"knative.dev/serving/pkg/network/status"
 	"strings"
 	"time"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/ingress.go [hunk 28]
```diff
@@ -19,6 +19,7 @@ package ingress
 import (
 	"context"
 	"fmt"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go]: Format Go code:
```suggestion
	"fmt"
	"reflect"

```

File: pkg/reconciler/ingress/ingress.go [hunk 29]
```diff
@@ -19,6 +19,7 @@ package ingress
...
+	"knative.dev/serving/pkg/network/status"
 	"reflect"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go [hunk 30]
```diff
@@ -0,0 +1,98 @@
...
+limitations under the License.
+*/
+
+package ingress
+
+import (
+	"context"
+	"fmt"
+	"knative.dev/net-kourier/pkg/config"
+	"knative.dev/pkg/system"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go [hunk 31]
```diff
@@ -0,0 +1,98 @@
...
+	"net/url"
+	"sort"
+	"strconv"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion
	"strconv"

	"knative.dev/net-kourier/pkg/config"
	"knative.dev/pkg/system"
```

File: pkg/reconciler/ingress/lister.go [hunk 32]
```diff
@@ -0,0 +1,98 @@
...
+		for _, rule := range ing.Spec.Rules {
+			for _, path := range rule.HTTP.Paths {
+				for _, host := range rule.Hosts {
+					targetURL, _ := url.ParseRequestURI(schema + "://" + host + path.Path)
+					urls = append(urls, targetURL)
+				}
+			}
+			target := status.ProbeTarget{
+				PodIPs:  sets.String{ip: sets.Empty{}},
+				URLs:    urls,
```
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion
				PodIPs: sets.String{ip: sets.Empty{}},
				URLs:   urls,
```

File: pkg/reconciler/ingress/lister.go [hunk 33]
```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2019 The Knative Authors.
```
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors.
```

File: copy_ksvc.sh (hunk 8 above)
Comment (mattmoor) [inline copy_ksvc.sh]: Remove trailing whitespace:
```suggestion

```

File: copy_ksvc.sh (hunk 9 above)
Comment (mattmoor) [inline copy_ksvc.sh]: Remove trailing whitespace:
```suggestion

```

File: copy_ksvc.sh (hunk 10 above)
Comment (mattmoor) [inline copy_ksvc.sh]: Remove trailing whitespace:
```suggestion

```

File: curl.stderr (hunk 11 above)
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
> GET / HTTP/1.1
```

File: curl.stderr (hunk 12 above)
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
> Host: testfoo56.default.127.0.0.1.nip.io
```

File: curl.stderr (hunk 13 above)
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
> User-Agent: curl/7.64.1
```

File: curl.stderr (hunk 14 above)
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
> Accept: */*
```

File: curl.stderr (hunk 15 above)
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
>
```

File: curl.stderr (hunk 16 above)
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
< HTTP/1.1 404 Not Found
```

File: curl.stderr (hunk 17 above)
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
< date: Sun, 29 Mar 2020 19:41:55 GMT
```

File: curl.stderr (hunk 18 above)
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
< server: envoy
```

File: curl.stderr (hunk 19 above)
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
< content-length: 0
```

File: curl.stderr (hunk 20 above)
Comment (mattmoor) [inline curl.stderr]: Remove trailing whitespace:
```suggestion
<
```

File: ready_timings.sh (hunk 21 above)
Comment (mattmoor) [inline ready_timings.sh]: Remove trailing whitespace:
```suggestion

```

File: ready_timings.sh (hunk 22 above)
Comment (mattmoor) [inline ready_timings.sh]: Remove trailing whitespace:
```suggestion

```

File: test.sh (hunk 23 above)
Comment (mattmoor) [inline test.sh]: Remove trailing whitespace:
```suggestion

```

File: pkg/generator/ingress_translator.go (hunk 24 above)
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion
	"fmt"
	"time"

```

File: pkg/generator/ingress_translator.go (hunk 25 above)
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/controller.go (hunk 26 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go]: Format Go code:
```suggestion
	"context"
	"strings"
	"time"

```

File: pkg/reconciler/ingress/controller.go (hunk 27 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/ingress.go (hunk 28 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go]: Format Go code:
```suggestion
	"fmt"
	"reflect"

```

File: pkg/reconciler/ingress/ingress.go (hunk 29 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go (hunk 30 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go (hunk 31 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion
	"strconv"

	"knative.dev/net-kourier/pkg/config"
	"knative.dev/pkg/system"
```

File: pkg/reconciler/ingress/lister.go (hunk 33 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors.
```

File: pkg/generator/ingress_translator.go (hunk 24 above)
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion
	"fmt"
	"time"

```

File: pkg/generator/ingress_translator.go [hunk 34]
```diff
@@ -18,6 +18,7 @@ package generator
...
```
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion

```

File: pkg/generator/ingress_translator.go (hunk 24 above)
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion
	"fmt"
	"time"

```

File: pkg/generator/ingress_translator.go (hunk 25 above)
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/controller.go (hunk 26 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go]: Format Go code:
```suggestion
	"context"
	"strings"
	"time"

```

File: pkg/reconciler/ingress/controller.go (hunk 27 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/ingress.go (hunk 28 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go]: Format Go code:
```suggestion
	"fmt"
	"reflect"

```

File: pkg/reconciler/ingress/ingress.go (hunk 29 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go (hunk 30 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go (hunk 31 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion
	"strconv"

	"knative.dev/net-kourier/pkg/config"
	"knative.dev/pkg/system"
```

File: pkg/reconciler/ingress/lister.go [hunk 35]
```diff
@@ -0,0 +1,99 @@
...
```
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors.
```

File: pkg/generator/ingress_translator.go (hunk 24 above)
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion
	"fmt"
	"time"

```

File: pkg/generator/ingress_translator.go (hunk 25 above)
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/controller.go (hunk 26 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go]: Format Go code:
```suggestion
	"context"
	"strings"
	"time"

```

File: pkg/reconciler/ingress/controller.go (hunk 27 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/ingress.go (hunk 28 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go]: Format Go code:
```suggestion
	"fmt"
	"reflect"

```

File: pkg/reconciler/ingress/ingress.go (hunk 29 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go [hunk 36]
```diff
@@ -0,0 +1,99 @@
...
+limitations under the License.
+*/
+
+package ingress
+
+import (
+	"context"
+	"fmt"
+	"knative.dev/net-kourier/pkg/config"
+	"knative.dev/pkg/system"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go [hunk 37]
```diff
@@ -0,0 +1,99 @@
...
+	"net/url"
+	"sort"
+	"strconv"
```
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion
	"strconv"

	"knative.dev/net-kourier/pkg/config"
	"knative.dev/pkg/system"
```

File: pkg/reconciler/ingress/lister.go (hunk 35 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors.
```

File: pkg/generator/ingress_translator.go (hunk 24 above)
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion
	"fmt"
	"time"

```

File: pkg/generator/ingress_translator.go (hunk 25 above)
Comment (mattmoor) [inline pkg/generator/ingress_translator.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/controller.go (hunk 26 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go]: Format Go code:
```suggestion
	"context"
	"strings"
	"time"

```

File: pkg/reconciler/ingress/controller.go (hunk 27 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/ingress.go (hunk 28 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go]: Format Go code:
```suggestion
	"fmt"
	"reflect"

```

File: pkg/reconciler/ingress/ingress.go (hunk 29 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go (hunk 36 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go (hunk 37 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go]: Format Go code:
```suggestion
	"strconv"

	"knative.dev/net-kourier/pkg/config"
	"knative.dev/pkg/system"
```

File: pkg/reconciler/ingress/lister.go (hunk 35 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go:2]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors.
```

File: pkg/generator/ingress_translator.go (hunk 24 above)
Comment (mattmoor) [inline pkg/generator/ingress_translator.go:20]: Format Go code:
```suggestion
	"fmt"
	"time"

```

File: pkg/generator/ingress_translator.go (hunk 25 above)
Comment (mattmoor) [inline pkg/generator/ingress_translator.go:22]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/controller.go (hunk 26 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:20]: Format Go code:
```suggestion
	"context"
	"strings"
	"time"

```

File: pkg/reconciler/ingress/controller.go (hunk 27 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/controller.go:24]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/ingress.go (hunk 28 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go:21]: Format Go code:
```suggestion
	"fmt"
	"reflect"

```

File: pkg/reconciler/ingress/ingress.go (hunk 29 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/ingress.go:23]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go (hunk 36 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go:23]: Format Go code:
```suggestion

```

File: pkg/reconciler/ingress/lister.go (hunk 37 above)
Comment (mattmoor) [inline pkg/reconciler/ingress/lister.go:26]: Format Go code:
```suggestion
	"strconv"

	"knative.dev/net-kourier/pkg/config"
	"knative.dev/pkg/system"
```

**Comment by jmprusi** [comment]
closed in favour of #18

**Pull request closed without merging**